config/htmlmin.php

```diff
@@ -46,4 +46,25 @@
 
     'force' => false,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Ignore Blade Files
+    |--------------------------------------------------------------------------
+    |
+    | Here you can specify paths, which you don't want to minimize, for example
+    | e-mails or markdowns.
+    |
+    */
+
+    'ignore' => [
+        'resources/views/emails',
+        'resources/views/html',
+        'resources/views/notifications',
+        'resources/views/markdown',
+        'resources/views/vendor/emails',
+        'resources/views/vendor/html',
+        'resources/views/vendor/notifications',
+        'resources/views/vendor/markdown',
```

Comment:
@GrahamCampbell Originally it was "resources/views/emails", will it be enough then? Just because without the others, Notification mails will be minified too, as those are stored in I guess the `html` folder. So `$user->notify()` will be broken as default (and I think it's quite general to use).